planner/optimizer/.gitignore

```diff
@@ -4,5 +4,4 @@ target/
 .classpath
 .project
 .metadata
-.settings/
-src/test/java/eu/seaclouds/platform/planner/optimizerTest/resources/output*
```

Comment:
@perezp There's no need to blind-trust me!. If you want to check my PR properly you should checkout my branch and try it yourself.

However, target is already included in your gitignore (https://github.com/SeaCloudsEU/SeaCloudsPlatform/blob/master/planner/optimizer/.gitignore#L3). Also, you don't need to specify the full path to ignore a file: http://stackoverflow.com/questions/1470572/gitignore-ignore-any-bin-directory.

However, thanks for reviewing it!